### ISSUE TITLE
Add Image Tooltip support for C# code and ImageSource Properties

### ISSUE THREAD

**Issue by ravero** [mfractor/mfractor-feedback#189]
The Image Tooltip tool could be applied to C# on hardcoded assignment of an ImageSource property. For instance, if we have the following XAML:

```xml
<Image x:Name="sampleImage" />
```

And the following code on the code-behind:

```csharp
sampleImage.Source = "image_name";
```

Considering that the ImageSource has automatic conversion of string values, we could have the Image Tooltip appearing when hovering over the image name.

The Intellisense to search for a file could also be applied in this context.

**Comment by matthewrdev** [comment]
This *should* be supported, however, it looks like there is now a bug in the project resolution when resolving the image assets.

This is fixed and will be fixed in MFractor 4.0

**Comment by matthewrdev** [comment]
Fix published in v0.5 for Windows and v3.10.4 for Mac.
